assignments/character_count.md

```diff
@@ -0,0 +1,18 @@
+##Character Count
+
+Given an array of colors, return a hash for each color with each character's character count. E.g.,
+colors = ["purple", "yellow", "green", "red"]
+
+green_hash = { "g": 1, "r": 1, "e": 2, "n": 1}
+red_hash = { "r": 1, "e": 1, "d": 1}
+purple_hash = { "p": 2, "e": 1, "u": 1, "l": 1, "r": 1}
+
+## Notes
+
+Remember, order doesn't matter in hashes.
```

Comment:
Never thought of it, but this is a good argument for not calling hashes dictionaries, at least from a learning standpoint.

Reply:
yeah, I definitely saw some openings for teaching OOP in this lesson, though I think will wait to go through the `on_bill` problem a little more before introducing.

Reply:
:true_story: Now to come up with a different analogy...